opts/mount.go

```diff
@@ -76,6 +76,9 @@ func (m *MountOpt) Set(value string) error {
 			case "volume-nocopy":
 				volumeOptions().NoCopy = true
 				continue
+			case "bind-nonrecursive":
+				bindOptions().NonRecursive = true
```

Comment:
Just noticed this (not introduced in this PR, but an existing issue);

> `bind-nonrecursive`: `true` or `false` (default)

Looks like `bind-nonrecursive=false` won't work, because we don't take the value into account (assuming the value will be `true`)

Probably better to fix that i a separate PR, but something we should look into